sensor/tests/connection/connection_test.go

```diff
@@ -16,6 +16,35 @@ var (
 	NginxDeployment1 = helper.K8sResourceInfo{Kind: "Deployment", YamlFile: "nginx.yaml", Name: "nginx-deployment"}
 )
 
+func Test_SensorHello(t *testing.T) {
+	if buildinfo.ReleaseBuild {
+		t.Skipf("Don't run test in release mode: feature flag cannot be enabled")
+	}
```

Comment:
I think there are 2 issues to address here:
- The value of `GetReconnect` should make sense independently of the feature flag
- If we keep this `if` here, we need a TODO annotation to remove it in the future (e.g., to a ticket about flipping the FF to true), as it may be easy to forget

Reply:
> The value of GetReconnect should make sense independently of the feature flag

For the `STARTUP`, yes. But we will only see `RECONNECT` if the FF is enabled, because otherwise sensor will just shutdown.